data/src/main/java/org/apache/iceberg/data/orc/GenericOrcReader.java

```diff
@@ -53,86 +50,13 @@ public GenericOrcReader(
 
   @Override
   public Record read(VectorizedRowBatch batch, int row) {
-    return (Record) reader.read(new StructColumnVector(batch.size, batch.cols), row);
+    return (Record) getReader().read(new StructColumnVector(batch.size, batch.cols), row);
```

Comment:
Getter methods should not start with `get`. It doesn't add anything (every method "gets" its return value) and is not idiomatic in other JVM languages.